src/Accounts/Authentication.ResourceManager/Models/PSAzureSubscription.cs

```diff
@@ -223,7 +223,11 @@ public string AuthorizationSource
         {
             get
             {
-               return this.GetTags();
+                if (!string.IsNullOrEmpty(this.GetProperty(AzureSubscription.Property.Tags)))
+                {
+                    return this.GetTags();
```

Comment:
We'd better check null in GetTags() instead of here. Considering additional work for updating Common libs, I'm totally fine with such temporary fix for OOB release, please fix it in common lib in coming release.

Reply:
> We'd better check null in GetTags() instead of here. Considering additional work for updating Common libs, I'm totally fine with such temporary fix for OOB release, please fix it in common lib in coming release.

Yes I will fix it in common and release it in next normal release